addons/stock/views/stock_picking_views.xml

```diff
@@ -366,7 +366,7 @@
                                     <field name="reserved_availability" string="Reserved" attrs="{'column_invisible': (['|','|', ('parent.state','=', 'done'), ('parent.picking_type_code', '=', 'incoming'), ('parent.immediate_transfer', '=', True)])}"/>
                                     <field name="quantity_done" string="Done" attrs="{'readonly': [('is_quantity_done_editable', '=', False)]}"/>
                                     <field name="product_uom" attrs="{'readonly': [('state', '!=', 'draft'), ('additional', '=', False)]}" options="{'no_open': True, 'no_create': True}" string="Unit of Measure" groups="uom.group_uom"/>
-                                    <button name="action_show_details" string="Register lots, packs, location" type="object" icon="fa-list" width="0.1"
+                                    <button name="action_show_details" type="object" icon="fa-list" width="0.1"
```

Comment:
The string is also set in https://github.com/odoo/odoo/blob/27f37d59aa4a7f8216a1bf8094278039fb1b423a/addons/stock/views/stock_move_views.xml#L125-L130

Reply:
done

Reply:
@Whenrow I didn't make this change, because the string is required when it has a "fa-list" class and the string will only be shown as tooltips.